recipes/emg3d/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "emg3d" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a027ccae7f13cbbde903aab9fbedb89da97c72af90ddb566ddec09c0a5f8311
+
+build:
+  skip: True    # [win]  # Remove when numba-feedstock/pull/37 is merged.
+  # noarch: python       # Enable when numba-feedstock/pull/37 is merged.
```

Comment:
You can leave the recipe as noarch even if windows is currently failing. noarch recipes get built on linux only, so as long as that passes, it is good.

Reply:
OK, I try. Last time the linter complained that `noarch` is not compatible with `skip`.

Reply:
Yeah just remove the skip statement too.